book/src/SDK/platform/authz.md

```diff
@@ -0,0 +1,16 @@
+## AuthZ
```

Comment:
Why AuthZ?

Reply:
Authz covers both Authentication and Authorization.  It's industry Jargon. I have no strong opinion either way TBH, but that's the explanation.

Reply:
Ah, nice TILT